proposals/0167-app-services.md

```diff
@@ -412,10 +412,11 @@ The `AppServiceManifest` is essentially detailing everything about a particular
 
 				
 		<param name="uriPrefix" type="String" mandatory="false">
-			<description> The URI prefix for this service. If provided, all PerformAppServiceInteraction requests must start with it.</description>
+			<description> The URI prefix for this service. If provided, all PerformAppServiceInteraction requests must start with it. This can also be referred to as the scheme of the URI. This scheme, if included, will be followed by a colon (:) as well as two slashes (//) if the service action includes a baseUri. If this is not included the uri shall start with a single slash (/). For example, uriPrefix://baseUri?parameters or uriPrefix:paramters or /baseUri?parameters </description>
 		</param>
-		<param name="uriScheme" type="JSONObject" mandatory="false">
-			<description> This is a custom schema for this service. SDL will not do any verification on this param past that it has a correctly formated JSON Object as its base. The uriScheme should contain all available actions to be taken through a PerformAppServiceInteraction request from an app service consumer. </description>
+
+		<param name="uriActionSheet" type="ServiceAction" array="true" mandatory="false">
+			<description> This is a custom schema for this service. Each of the actions should be formatted into a ServiceAction object that contains basic information to ensure consumers can read the data. The uriActionSheet should contain all available actions to be taken through a PerformAppServiceInteraction request from an app service consumer. If the app intends to expose Quick Action actions, the list should be ordered with the most important at index 0. </description>
```

Comment:
Suggestion update: 

If the app intends to expose '**Quick Action'** actions, the list should be ordered with the most important at index 0. </description>